test/e2e/content/case.turned-off-for-host.test.js

```diff
@@ -0,0 +1,141 @@
+            await popup.waitForSelector(selectors.showChangesCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should not show the 'Show highlights' checkbox", async function() {
+            await popup.waitForSelector(selectors.showHighlightsCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should show the 'Reload page' button", async function() {
+            await popup.waitForSelector(selectors.reloadPage, {
+                hidden: false
+            });
+        });
+    });
+
+    describe("When the user opens or reloads the page, it", function() {
+        before(async function() {
+            // Open page and popup just so we can access the doNotReplaceList
+            options = await browser.newPage();
+            await options.goto(optionsURL);
+            page = await browser.newPage();
+            await page.goto(testURL + text);
+            popup = await browser.newPage();
+            await popup.goto(popupURL);
+
+            // Turn off replacements, reload popup to simulate re-opening it
+            await storage.set(options, {
+                doNotReplaceList: [new URL(testURL).host]
+            });
+            await page.reload();
+            await popup.reload();
+        });
+
+        after(async function() {
+            await storage.clear(popup);
+
+            await popup.close();
+            await page.close();
+            await options.close();
+        });
+
+        it("should still say 'he' and 'she'", async function() {
+            const bodyText = await page.$eval("body", e => e.innerText);
+            expect(bodyText).to.include(" he ");
+            expect(bodyText).to.include(" she ");
+        });
+
+        it("should not have any insertions or deletions", async function() {
+            expect(await page.$$(selectors.ins)).to.be.empty;
+            expect(await page.$$(selectors.del)).to.be.empty;
+        });
+
+        it("should explain in the popup", async function() {
+            const statusText = await popup.$eval(
+                selectors.status,
+                e => e.innerText
+            );
+            expect(statusText).to.equal(
+                "You've turned off Degender the Web for this page."
+            );
+        });
+
+        it("should not show the 'Show changes' checkbox", async function() {
+            await popup.waitForSelector(selectors.showChangesCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should not show the 'Show highlights' checkbox", async function() {
+            await popup.waitForSelector(selectors.showHighlightsCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should not show the 'Reload page' button", async function() {
+            await popup.waitForSelector(selectors.reloadPage, { hidden: true });
+        });
+    });
+});
```

Comment:
Missing test: The "allow replacements on host" checkbox should be unchecked.

Reply:
I've now added the corresponding `turned-on` test as well.